src/handlebars/partials/footer.handlebars

```diff
@@ -25,16 +25,8 @@
   <div class="margin-auto" id="footer-social-bar">{{> social-bar }}</div>
 
   <p style="line-height: 1rem;">Java and OpenJDK are trademarks or registered trademarks of Oracle and/or its affiliates. Other names may be trademarks of their respective owners.</p>
-
-  <!--
-  <div style="margin-top: 1.5rem;" id="google_translate_element"></div>
-  <script type="text/javascript">
-    function googleTranslateElementInit() {
-      new google.translate.TranslateElement({pageLanguage: 'en', includedLanguages: 'en,pt,zh-CN', layout: google.translate.TranslateElement.InlineLayout.SIMPLE}, 'google_translate_element');
-    }
-  </script>
-  <script type="text/javascript" src="https://translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
-  -->
+  <br>
+  <p style="line-height: 1rem;"><strong>Spotted an issue with the website?</strong><br><a target="_blank" class="light-link" href="https://github.com/AdoptOpenJDK/openjdk-website/issues">Raise a GitHub issue</a> or <a target="_blank" class="light-link" href="https://github.com/AdoptOpenJDK/openjdk-website/blob/master/CONTRIBUTING.md">contribute a change!</a></p>
```

Comment:
@karianna What's your thoughts on the footer on [staging](https://staging.adoptopenjdk.net)? I'm not sure if it's an issue, or whether slightly longer footers are pretty standard?

I'm trying to think of an alternative place to put the link on every page (already adding it to the Get Involved page, but that's not enough)... no ideas just yet!